packages/core/src/browser/widgets/navigatable.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import URI from '../../common/uri';
+
+export interface Navigatable {
```

Comment:
move it to `core/src/browser` and reexport in `index.ts`

Reply:
ok